src/js/discharge-wizard/containers/GuidancePage.jsx

```diff
@@ -331,21 +331,34 @@ class GuidancePage extends React.Component {
           </ul>
         </div>
         <h4>Additional Resources</h4><hr/>
-        <ul>
-          <li><a target="_blank" href="https://www.vets.gov/health-care/health-conditions/military-sexual-trauma/">VA health benefits for Veterans who experience military sexual trauma</a></li>
-          <li><a target="_blank" href="https://www.vets.gov/health-care/health-conditions/mental-health/">VA health benefits for Veterans with mental health conditions</a></li>
-          <li><a target="_blank" href="https://www.vets.gov/health-care/health-conditions/mental-health/ptsd/">VA health benefits for Veterans with PTSD</a></li>
-          {this.props.formValues['1_branchOfService'] === 'army' && <li><a target="_blank" href="http://arba.army.pentagon.mil/adrb-overview.html">Army Discharge Review Board</a></li>}
-          {['navy', 'marines'].includes(this.props.formValues['1_branchOfService']) && <li><a target="_blank" href="http://www.secnav.navy.mil/mra/CORB/pages/ndrb/default.aspx">Navy Discharge Review Board</a></li>}
-          <li><a target="_blank" href="https://www.benefits.va.gov/BENEFITS/docs/COD_Factsheet.pdf">VA Guidance on Character of Discharge Reviews</a></li>
-        </ul>
+        {this.renderAdditionalResources()}
         <p>
           <strong>Please note:</strong> This information was created based on how you answered the questions on the previous page. This information will not be specific to someone with different answers to the questions.
         </p>
       </div>
     );
   }
 
+  renderAdditionalResources() {
+    const serviceBranch = this.props.formValues['1_branchOfService'];
+
+    return (
+      <ul>
+        <li><a target="_blank" href="https://www.vets.gov/health-care/health-conditions/military-sexual-trauma/">VA health benefits for Veterans who experience military sexual trauma</a></li>
+        <li><a target="_blank" href="https://www.vets.gov/health-care/health-conditions/mental-health/">VA health benefits for Veterans with mental health conditions</a></li>
+        <li><a target="_blank" href="https://www.vets.gov/health-care/health-conditions/mental-health/ptsd/">VA health benefits for Veterans with PTSD</a></li>
+        <li><a target="_blank" href="https://www.benefits.va.gov/BENEFITS/docs/COD_Factsheet.pdf">VA Guidance on Character of Discharge Reviews</a></li>
+        {branchOfService === 'army' && <li><a target="_blank" href="http://arba.army.pentagon.mil">Army Review Boards Agency</a></li>}
```

Comment:
I think you mean `serviceBranch` in these conditionals, since that's what you defined above.

Reply:
good catch, thanks!